components/Header/Header.tsx

```diff
@@ -181,9 +177,6 @@ const Header: React.FunctionComponent<HeaderProps> = ({
         <Text style={{ fontSize: 8, color: colors.border }}>
           {'(' + dimensions.width + 'x' + dimensions.height + ')-' + dimensions.scale}
         </Text>
-        <Text style={{ fontSize: 7, color: colors.border }}>
-          {dimensions.orientation === 'landscape' ? translate('info.landscape') : translate('info.portrait')}
-        </Text>
```

Comment:
ok, we can rid of it as well... this could make sense in Test, but now, I don't think so.